Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Haptics/SimpleXRRumble.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using UnityEngine.Experimental.Input.Plugins.XR;
+using UnityEngine;
+
+namespace UnityEngine.Experimental.Input.Plugins.XR.Haptics
+{
+    public class SimpleXRRumble
```

Comment:
Why not just a struct?

Reply:
I'm not really passing this thing around, and I wanted to have a guaranteed InputDevice reference via the Constructor.  I wasn't a big fan of DualMotorRumble always requiring the device again and again when setting or updating.

Reply:
You could just pass the InputDevice to the struct constructor. IMO adding a separate heap object for what should just be part of a class isn't a good thing.